release_monitor/release_monitor.py

```diff
@@ -113,11 +115,13 @@ def create_package_from_pypi_dict(dict):
         list_of_pypi_updates = feedparser.parse(self.pypi_url + "rss/updates.xml").entries
         try:
             updated_packages = set(map(create_package_from_pypi_dict, list_of_pypi_updates))
-        except KeyError:
+        except KeyError as e:
             # if the "title" does not exist, catch the error and return nothing
+            logger.error('Titles does not exist : {e}'.format(e=str(e)))
```

Comment:
What is the point of logging these errors here?

Reply:
And by here I mean in the whole file :)

Reply:
@msrb As I mentioned below, in case upstream format change, those logger.errors are beneficial to the developers to correct their code displayed in Sentry. If you think otherwise, we can make those into logger.info so that Sentry will not create event for that.